code/modules/cargo/bounties/medical.dm

```diff
@@ -46,9 +46,9 @@
 	required_count = 3
 	wanted_types = list(/obj/item/organ/tongue)
 
-/datum/bounty/item/medical/lizard_tail
+/*/datum/bounty/item/medical/lizard_tail
```

Comment:
Is it worth removing the whip bounty as well if we remove just the tail on its own?

Reply:
We were giving Unathi the cronch last I checked which would make this unattainable.